source/docs/advanced/theme/route-dispatcher.md

```diff
@@ -0,0 +1,134 @@
+---
+id: route-dispatcher
+title: Handle dynamic URLs with the Dispatcher
+---
+
+In some cases, you will need a deeper customization of URLs than what you can do in the [Add a new page guide](../../essentials/add-a-page-client-side). For instance, for some SEO reasons, you may prefer to have `/my-product` instead of `/product/my-product-slug`.
```

Comment:
```suggestion
In some cases, you will need more control over URLs formats than what we introduced in the [Add a new page guide](../../essentials/add-a-page-client-side). For instance, you may prefer to have `/my-product` instead of `/product/my-product-slug` for SEO reasons.
```